Software/app/secure_element_lorawan/lib_dependency/Commissioning.h

```diff
@@ -0,0 +1,80 @@
+/*!
+ * \file      Commissioning.h
+ *
+ * \brief     End-device commissioning parameters
+ *
+ * \copyright Revised BSD License, see section \ref LICENSE.
+ *
+ * \code
+ *                ______                              _
+ *               / _____)             _              | |
+ *              ( (____  _____ ____ _| |_ _____  ____| |__
+ *               \____ \| ___ |    (_   _) ___ |/ ___)  _ \
+ *               _____) ) ____| | | || |_| ____( (___| | | |
+ *              (______/|_____)_|_|_| \__)_____)\____)_| |_|
+ *              (C)2013-2020 Semtech
+ *
+ * \endcode
+ */
+/**
+  ******************************************************************************
+  *
+  *          Portions COPYRIGHT 2020 STMicroelectronics
+  *
+  * @file    Commissioning.h
+  * @author  MCD Application Team
+  * @brief   End-device commissioning parameters
+  ******************************************************************************
+ */
+/* Define to prevent recursive inclusion -------------------------------------*/
+#ifndef __COMMISSIONING_H__
+#define __COMMISSIONING_H__
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+/*!
+ ******************************************************************************
+ ********************************** WARNING ***********************************
+ ******************************************************************************
+
+ The LoRaWAN AES128 keys are stored and provisionned on secure-elements.
+
+ This project providdes a software emulated secure-element.
+ The LoRaWAN AES128 keys SHALL be updated under
+ src/peripherals/<secure-element name>-se\se-identity.h file.
+
+ ******************************************************************************
+ ******************************************************************************
+ ******************************************************************************
+ */
```

Comment:
I might be wrong, but this warning doesn't apply for this particular app right?

Reply:
\+ minor spelling for all Commissionings: provissioned & provides

Reply:
Done, fixed